pkg/db/db_test.go

```diff
@@ -25,8 +25,10 @@ import (
 	"os"
 	"os/exec"
 	"path"
+	"slices"
```

Comment:
for the commit message, I would add something like this:
"this will translate in improved performance. And in most cases, most apps will have deployments"